selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/CreateWorkspace.java

```diff
@@ -252,4 +252,34 @@ public void clickOnSingleMachineTab() {
   public void clickOnMultiMachineTab() {
     redrawUiElementsTimeout.until(visibilityOf(selectMultiMachineStacksTab)).click();
   }
+
+  public void waitWorkspaceIsCreatedDialogIsVisible() {
+    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
+        .until(visibilityOfElementLocated(By.xpath(Locators.WORKSPACE_CREATED_DIALOG)));
+    WaitUtils.sleepQuietly(1);
+  }
+
+  public void clickOnEditWorkspaceButton() {
+    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
+        .until(elementToBeClickable(By.xpath(Locators.EDIT_WORKSPACE_DIALOG_BUTTON)))
+        .click();
+  }
+
+  public void clickOnOpenInIDEButton() {
+    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
+        .until(elementToBeClickable(By.xpath(Locators.OPEN_IN_IDE_DIALOG_BUTTON)))
+        .click();
+  }
+
+  public void clickOnCreateButtonAndStartWorkspace() {
```

Comment:
According to the command inside, the method name should be "clickOnCreateButtonAndOpenInIDE"